src/user/mod.rs

```diff
@@ -33,18 +32,20 @@ impl UserMetadata {
             .get_or_insert_with(|| parser::get_inputs(root_ref.unwrap()))
     }
 
-    pub fn get_inputs(&mut self) -> HashMap<String, NixNode> {
+    pub(crate) fn get_inputs(&mut self) -> HashMap<String, NixNode> {
         self.ensure_inputs().clone()
     }
```

Comment:
Is this method even needed anymore?

Reply:
Not sure

Reply:
last time I checked, cargo mentioned it as `unused`.